packages/camera/camera_windows/test/camera_windows_test.dart

```diff
@@ -181,6 +181,11 @@ void main() {
             verify(mockApi.dispose(captureAny));
         expect(verification.captured[0], cameraId);
       });
+
+      test('Should report support for image streaming', () async {
+        final CameraWindows plugin = CameraWindows(api: MockCameraApi());
+        expect(plugin.supportsImageStreaming(), true);
+      });
     });
 
```

Comment:
`onStreamedFrameAvailable` also needs unit testing, validating that listening to the stream and canceling the subscription make the expected calls.

Reply:
Added a test case for listening. Cancellation is handled on the native side, so I don’t think it can be tested here using the mocks.